package.json

```diff
@@ -12,17 +12,17 @@
     "lint": "tslint --project ./tsconfig.json"
   },
   "dependencies": {
-    "cac": "^6.5.6",
-    "postcss": "^7.0.26",
-    "tailwindcss": "^1.2.0"
+    "cac": "^6.6.1",
+    "postcss": "^7.0.32",
+    "tailwindcss": "1.2.0"
```

Comment:
I had to fix the version here. See https://github.com/scoville/tailwind-generator/issues/18 for more.